Document DeviceEnroller.exe behavior when local admin already exists (#6149)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/policy-csp-restrictedgroups.md

<!--Description-->
This security setting allows an administrator to define the members of a security-sensitive (restricted) group. When a Restricted Groups Policy is enforced, any current member of a restricted group that is not on the Members list is removed. Any user on the Members list who is not currently a member of the restricted group is added. You can use Restricted Groups policy to control group membership. Using the policy, you can specify what members are part of a group. Any members that are not specified in the policy are removed during configuration or refresh. For example, you can create a Restricted Groups policy to only allow specified users (for example, Alice and John) to be members of the Administrators group. When policy is refreshed, only Alice and John will remain as members of the Administrators group.  

> Please note that DeviceEnroller.exe will not elevate the user if there is already a local admin group pre-configured on the device. This is a security measure in the executable where it checks for other non-disabled Administrators membership and if there are already at least one then the tool exits without elevating.

Reviewer: Turn this "Please note" notation into a MarkDown Note blob (colored HTML Note blob on the docs.microsoft.com pages) :
```suggestion
> [!NOTE]
> DeviceEnroller.exe will not elevate the user if a pre-configured local admin group already exists on the device. This is a security measure in the executable where it checks for other non-disabled administrators' membership(s). If at least one already exists, the tool will exit without elevating.
```
Please note that even though my English skills are near-native, it is not my primary language. There is a possibility that some of these changes might be slightly imprecise when it comes to conveying the importance of the individual parts of the sentences, so you may want to evaluate my suggestion and maybe even edit the suggestion before committing it to the PR (or wait for additional suggestions from the technical writers for the MS Docs team).
